person.rb

```diff
@@ -6,6 +6,7 @@
 class Person < Nameable
   attr_reader :id
   attr_accessor :name, :age
+  attr_reader :rentals
```

Comment:
- [ ] Getters and Setters for @rentals should be present, the `rentals` should be `attr_accessor` kindly fix that 🙏🏾